packages/components/src/DatePicker/DatePicker.css.d.ts

```diff
@@ -1,4 +1,5 @@
 declare const styles: {
+  readonly "react-datepicker__month--selecting-range": string;
```

Comment:
Was curious where did this come from. It was just some nested `scss` sillyness

Normall nested SCSS

https://github.com/Hacker0x01/react-datepicker/blob/e791571c2e8ba694b2080dab8ba49758844bc434/src/stylesheets/datepicker.scss#L483-L488

getting nested under `:global`

https://github.com/Hacker0x01/react-datepicker/blob/master/src/stylesheets/datepicker-cssmodules.scss

and because it's getting written as `.react-datepicker__month--selecting-range & ` it will be added before `:global`.

![image](https://user-images.githubusercontent.com/15986172/146575198-0f586374-b60a-4d80-8158-5d8cadacbd59.png)

SCSS Protip: Look at the `&`

```scss
.foo {
  $.bar {}
}

/* output: .foo.bar {} */

.foo {
  .bar {}
}

/* output: .foo .bar {} */

.foo {
  .bar & {}
}

/* output: .bar .foo {} */
```